code/modules/paperwork/ticketmachine.dm

```diff
@@ -208,6 +208,11 @@
 	var/obj/machinery/ticket_machine/source
 	var/ticket_number
 
+/obj/item/ticket_machine_ticket/attack_self(mob/user)
+	if(Adjacent(user))
+		user.visible_message("<span class='notice'>[user] shows you: \the [src.name].</span>", "<span class='notice'>You show \the [src.name].</span>")
+	add_fingerprint(user)
```

Comment:
Great idea to add fingerprints, but the placement feels a bit funny, only leaving fingerprints when showing it to someone? I don't even know if it's possible to add them when picking up.

Reply:
im pretty sure all items do that already

Reply:
does ID-showing also use a colon?

Reply:
I just copypasted this code from the ID's attack_self, my only change was removing the icon that is shown of the object that is shown because the maptext which gives tickets their number wasn't appearing.

I'll have to check in-game whether it already has your fingerprints at this point or not. You'd think so.